python/paddle/fleet/base/role_maker.py

```diff
@@ -101,6 +101,14 @@ def server_index(self):
         """
         raise NotImplementedError("Please implement this method in child class")
 
+    def node_num(self):
+        """
+        Get the training node number
+        Returns:
+            int: node num
+        """
+        raise NotImplementedError("Please implement this method in child class")
```

Comment:
what is the difference between `node_num` and `worker_num`

Reply:
> what is the difference between `node_num` and `worker_num`

For the machine with muti-GPU, the `node num` stands for the number of machines, and the `worker_num` stands for the number of GPU cards.